google-cloud-spanner/src/main/java/com/google/cloud/spanner/Type.java

```diff
@@ -204,13 +210,23 @@ private Type(
       Code code,
       @Nullable Type arrayElementType,
       @Nullable ImmutableList<StructField> structFields) {
+    this.proto = null;
```

Comment:
Could we always populate the corresponding proto type?

Reply:
That's a little difficult. This constructor is used to create the fixed type constants for all known types in the client library. They do not have a proto definition available at construction time. See for example here: https://github.com/googleapis/java-spanner/blob/f637f1e5b595c97f4355ac7ab147aa0d261d20a5/google-cloud-spanner/src/main/java/com/google/cloud/spanner/Type.java#L49